src/chrome/content/rules/Images-Amazon.com.xml

```diff
@@ -0,0 +1,58 @@
+<!--
+	Invalid certificate:
+		ec2.images-amazon.com
+		ec4.images-amazon.com
+		ec5.images-amazon.com
+		ec6.images-amazon.com
+		ec8.images-amazon.com
+		ecx.images-amazon.com
+		g-ecx.images-amazon.com
+	
+	Time out:
+		ec1.images-amazon.com
+		ec3.images-amazon.com
+
+	https://trac.torproject.org/projects/tor/ticket/7539:
+		ecx.images-amazon.com
+
+-->
+<ruleset name="Amazon-Images.com">
+
+	<target host="ec1.images-amazon.com" />
+	<target host="ec2.images-amazon.com" />
+	<target host="ec3.images-amazon.com" />
+	<target host="ec4.images-amazon.com" />
+	<target host="ec5.images-amazon.com" />
+	<target host="ec6.images-amazon.com" />
+	<target host="ec8.images-amazon.com" />
+	<target host="g-ecx.images-amazon.com" />
+	<target host="z-ak.images-amazon.com" />
+
+	<!-- Zeitgeist MP3 player https://trac.torproject.org/projects/tor/ticket/7594 -->
+	<exclusion pattern="^http://g-ecx\.images-amazon\.com/.*(?:[mM][pP]3|zeitgeist|digital/music)" />
```

Comment:
Can we get rid of the non-capturing group here?

Reply:
It seems that this regex will match more or less everything :/
Are there any up to date examples to verify this bug? Most Amazon services are now HTTPS by default.

Reply:
After reviewing the Tor tracker ticket, it looks that the previously broken functionality no longer exists.